app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -1,101 +1,59 @@
 import React, { useState, useEffect } from "react";
 
-import EmptyNotesListImage from "images/EmptyNotesList";
-import { Delete } from "neetoicons";
 import { Button, PageLoader } from "neetoui";
-import { Container, Header, SubHeader } from "neetoui/layouts";
+import { Container, Header } from "neetoui/layouts";
 
-import notesApi from "apis/notes";
-import EmptyState from "components/Common/EmptyState";
-
-import DeleteAlert from "./DeleteAlert";
+import { NOTES } from "./constants";
+import NotesList from "./NotesList";
+import NotesMenu from "./NotesMenu";
 import NewNotePane from "./Pane/Create";
-import Table from "./Table";
 
 const Notes = () => {
   const [loading, setLoading] = useState(true);
   const [showNewNotePane, setShowNewNotePane] = useState(false);
-  const [showDeleteAlert, setShowDeleteAlert] = useState(false);
   const [searchTerm, setSearchTerm] = useState("");
-  const [selectedNoteIds, setSelectedNoteIds] = useState([]);
-  const [notes, setNotes] = useState([]);
+  const [notes, setNotes] = useState([{}]);
```

Comment:
Keep the initial value as an empty array. Keeping it as an empty array with empty object doesn't make sense until you add the keys of the object with a default value.

Reply:
I was encountering an ESlint error initially on keeping it as an empty array which is why I resorted to using an array with one object element.

`Argument of type '{ id: number; title: string; description: string; tag: string; status: string; created_by: string; created_at: string; avatar: any; }[]' is not assignable to parameter of type 'SetStateAction<never[]>'.
  Type '{ id: number; title: string; description: string; tag: string; status: string; created_by: string; created_at: string; avatar: any; }[]' is not assignable to type 'never[]'.
    Type '{ id: number; title: string; description: string; tag: string; status: string; created_by: string; created_at: string; avatar: any; }' is not assignable to type 'never'. `

Reply:
That's not an Eslint error from my understanding. It's a TypeScript error in VSCode, which you shouldn't be bothered about. Like I had already mentioned before, source of truth is the CLI. If you run eslint from terminal and if this issue pops up, then we should be bothered about it.